.github/workflows/dist_release.yml

```diff
@@ -1,5 +1,5 @@
 # Updates the "dev" github release whenever new changes are merged into master.
-# This provides an up to date tarball that https://github.com/ansible/ansible_catalog can use.
+# This provides an up to date tarball that https://github.com/ansible/automation_services_catalog can use.
```

Comment:
This is going to change to be whatever we decide the community name to be, just to keep in mind. Also right now it's still actually `ansible-catalog` (with a - instead of an _, so it was wrong anyway)